spec/index.bs

```diff
@@ -6640,7 +6644,46 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
         or the third item of the sequence.
 </div>
 
-# Temporary usages of non-exported dfns # {#temp-dfn-usages}
+# Appendix # {#appendix}
+
+## Texture Format Capabilities ## {#texture-format-caps}
+
+<table class='formats'>
+    <tr>
+        <th>Format
+        <th>Block
+        <th>Component
+        <th>Supported Usage
+        <th>Filter
+    <tr>
+        <td>{{GPUTextureFormat/"r8unorm"}}
+        <td>
+        <td>{{GPUTextureComponentType/"float"}}
+        <td>{{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_DST}}, {{GPUTextureUsage/SAMPLED}}, {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
+        <td><span>&#10003;</span>
```

Comment:
nit: apparently can be `&checkmark;`.

curious, why is the span needed?

Reply:
mostly because I don't know what I'm doing 😆